lpwan-backend/pom.xml

```diff
@@ -113,6 +113,11 @@
             <artifactId>device-capability-model</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-csv</artifactId>
```

Comment:
Was this being used already some where in repo?

Reply:
This is a new dependency we have bought in. We will add the dependency to the mapper properties file in core. And will create a PR for that